source/SpinalHDL/Sequential logic/registers.rst

```diff
@@ -7,20 +7,20 @@ Registers
 Introduction
 ------------
 
-| Creating register is very different than VHDL/Verilog.
-| In Spinal, you don't have process/always blocks. Registers are explicitly defined at the declaration.
-| This difference against traditional event driven HDL has a big impact :
+| Creating registers in SpinalHDL is very different than in VHDL or Verilog.
+| In Spinal, there are no process/always blocks. Registers are explicitly defined at declaration.
+| This difference from traditional event driven HDL has a big impact:
 
 
-* You can assign registers and wires in the same scope, you don't have to split your code between process/always blocks
+* You can assign registers and wires in the same scope, meaning the code doesn't need to be split between process/always blocks
 * It make things much more flexible (see :ref:`Functions <function>` )
 
-The way how clock and reset wire are managed are explained in the :ref:`Clock domain <clock_domain>` chapter.
+Clocks and resets are handled separately, see the :ref:`Clock domain <clock_domain>` chapter for details.
```

Comment:
The meaning of the phrase is changed here, so I evoke @Dolu1990 to comment on this change.

Reply:
after re-reading i think this is fine